common/app/views/fragments/commercial/cardLogo.scala.html

```diff
@@ -0,0 +1,10 @@
+@(branding: common.commercial.Branding, isStandardSizeCard: Boolean)(implicit request: RequestHeader)
+
+<div class="badge @if(!isStandardSizeCard) {badge--branded}">
+    @branding.label
+    <a class="badge__link" href="@branding.sponsorLink" data-link-name="logo link">
+        <img src="@branding.sponsorLogo.url"
+             alt="@{branding.sponsorName}'s logo"
```

Comment:
I see.  I always thought alt was to tell you what you were missing if you couldn't see the image.  I'll change that.